pkg/agent/plugin/workloadattestor/docker/cgroup/dockerfinder.go

```diff
@@ -101,7 +108,7 @@ func (f *containerIDFinder) FindContainerID(cgroup string) (string, bool) {
 	if len(matches) == 0 {
 		return "", false
 	}
-	return matches[submatchIndex], true
+	return string(matches[submatchIndex]), true
```

Comment:
I don't think that converting to string is needed.

Reply:
Indeed the conversion is not needed. I was working on code from 0.9.0 tag that has this conversion. I just noticed that this was removed in master branch (I guess also removed from 0.10.0 release). I will fix this - maybe a rebase on master branch should do the trick